modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -4,36 +4,52 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, requirements, jdk, installation, jre
 
-Recommended minimum hardware is:
+If you plan to install Mule and run it on premises, review these minimum hardware and software requirements before you install.
 
-* 2GHz CPU or 1 virtual CPU in virtualized environments
-* 1GB of RAM
-* 4GB of storage
+== Minimum Hardware Requirements
+
+* 2GHz CPU or 1 virtual CPU in virtualized environments.
+* 1GB of RAM.
+* 4GB of storage.
 
 Adjust RAM to match your latency requirements and the size and number of simultaneous messages that applications process.
 
-Software requirements are:
+== Required Software
+
+Verify that you use a supported version of Java before you install Mule.
+
+[%header,cols="20a,75a"]
+|===
+|Software |Version
+|JDK | JDK 1.8.0 (1.8.0_202 recommended), IBM JDK 1.8, OpenJDK 8 (latest update recommended)
+|===
+
+== Supported Software
+
+[NOTE]
+Mule runs on any operating system that runs a supported Java Runtime Environments. Mule runtime is tested against the following software and operating system resulting in a valid and correct functionality.
```

Comment:
Java Runtime Environments. -> Java Runtime Environment (JRE).

Reply:
"Mule runtime" -> "Mule" or "The Mule runtime engine"

Reply:
"operating system" -> "operating systems"

"resulting in a valid and correct functionality" seems unnecessary. If we really need to point out that we did functional testing (vs. some other, such as performance testing) and that it passed, we need to reword a bit. Maybe something like, "Mule passed functional testing against the following... " or some such, but I think that passing is implied without "resulting in a valid and correct functionality"  at the end.

Reply:
Updated.